src/theory/quantifiers/fmf/bounded_integers.cpp

```diff
@@ -407,21 +413,18 @@ void BoundedIntegers::checkOwnership(Node f)
             Trace("bound-int") << "Variable " << v << " is bound because of int range literals " << bound_lit_map[0][v] << " and " << bound_lit_map[1][v] << std::endl;
           }
         }else if( it->second==BOUND_SET_MEMBER ){
-          // only handles infinite element types currently (cardinality is not
-          // supported for finite element types #1123). Regardless, this is
-          // typically not a limitation since this variable can be bound in a
-          // standard way below since its type is finite.
-          if (!d_env.isFiniteType(v.getType()))
```

Comment:
Do we support set.card for finite types?

Reply:
We are incomplete for set.card with finite types.  This PR is making it so that `set.card` is not introduced in `fmf-bound`, so this is a non-issue.